Freshli.Test/Unit/Python/VersionMatcherTest.cs

```diff
@@ -256,68 +273,67 @@ string secondChildBaseVersion
     [InlineData("1.0.0", ">=0.0.1", true)]
     [InlineData("1.0.0", ">=0.1.0", true)]
     [InlineData("1.0.0", ">=1.0.0", true)]
-    [InlineData("1.0.0", ">=1.0.0-alpha", true)]
+    [InlineData("1.0.0", ">=1.0.0a1", true)]
     [InlineData("1.0.0", ">=1.0.1", false)]
     [InlineData("1.0.0", ">=1.1.0", false)]
     [InlineData("1.0.0", ">=2.0.0", false)]
     [InlineData("1.0.0", "~=0.0.1", false)]
     [InlineData("1.0.0", "~=0.1.0", false)]
-    [InlineData("1.0.0", "~=1", true)]
     [InlineData("1.0.0", ">=1", true)]
     [InlineData("1.0.0", "~=1.0.0", true)]
-    [InlineData("1.0.0", "~=1.0.0-alpha", true)]
+    [InlineData("1.0.0", "~=1.0.0a1", true)]
     [InlineData("1.0.0", "~=1.0.1", false)]
-    [InlineData("1.0.0", "~=1.0.1-alpha1", false)]
+    [InlineData("1.0.0", "~=1.0.1a1", false)]
     [InlineData("1.0.0", "~=1.1", false)]
     [InlineData("1.0.0", "~=1.1.0", false)]
-    [InlineData("1.0.0", "~=1.1.0-alpha1", false)]
+    [InlineData("1.0.0", "~=1.1.0a1", false)]
     [InlineData("1.0.0", "~=2", false)]
     [InlineData("1.0.0", "~=2.0", false)]
     [InlineData("1.0.0", "~=2.0.0", false)]
-    [InlineData("1.0.0-alpha", "!=1.0.0", true)]
-    [InlineData("1.0.0-alpha", "<1.0.0", true)]
-    [InlineData("1.0.0-alpha", "<=1.0.0", true)]
-    [InlineData("1.0.0-alpha", ">1.0.0", false)]
-    [InlineData("1.0.0-alpha", ">=1.0.0", false)]
-    [InlineData("1.0.0-alpha", "~=1.0.0", false)]
-    [InlineData("1.0.0-alpha+test1", "!=1.0.0-alpha+test2", false)]
-    [InlineData("1.0.0-alpha+test1", "<1.0.0-alpha+test2", false)]
-    [InlineData("1.0.0-alpha+test1", "<=1.0.0-alpha+test2", true)]
-    [InlineData("1.0.0-alpha+test1", "==1.0.*", true)]
-    [InlineData("1.0.0-alpha+test1", ">1.0.0-alpha+test2", false)]
-    [InlineData("1.0.0-alpha+test1", ">=1.0.0-alpha+test2", true)]
-    [InlineData("1.0.0-alpha+test1", "~=1.0.0-alpha+test2", true)]
-    [InlineData("1.0.0-alpha+test2", "!=1.0.0-alpha+test1", false)]
-    [InlineData("1.0.0-alpha+test2", "<1.0.0-alpha+test1", false)]
-    [InlineData("1.0.0-alpha+test2", "<=1.0.0-alpha+test1", true)]
-    [InlineData("1.0.0-alpha+test2", "==1.0.*", true)]
-    [InlineData("1.0.0-alpha+test2", ">1.0.0-alpha+test1", false)]
-    [InlineData("1.0.0-alpha+test2", ">=1.0.0-alpha+test1", true)]
-    [InlineData("1.0.0-alpha+test2", "~=1.0.0-alpha+test1", true)]
-    [InlineData("1.0.0-alpha1", "~=1.0.1-alpha1", false)]
-    [InlineData("1.0.0-alpha1", "~=1.0.1-alpha2", false)]
-    [InlineData("1.0.0-alpha1", "~=1.1.0-alpha1", false)]
-    [InlineData("1.0.0-alpha1", "~=1.1.0-alpha2", false)]
-    [InlineData("1.0.0-alpha2", "~=1.0.1-alpha1", false)]
-    [InlineData("1.0.0-alpha2", "~=1.1.0-alpha1", false)]
+    [InlineData("1.0.0a1", "!=1.0.0", true)]
+    [InlineData("1.0.0a1", "<1.0.0", true)]
+    [InlineData("1.0.0a1", "<=1.0.0", true)]
+    [InlineData("1.0.0a1", ">1.0.0", false)]
+    [InlineData("1.0.0a1", ">=1.0.0", false)]
+    [InlineData("1.0.0a1", "~=1.0.0", false)]
```

Comment:
@mscottford In reviewing the use cases below with an updated expected result, it seemed to me that the original expected results were incorrect, but I definitely might not be understanding the logic used by the version specifier correctly. Let me know your thoughts...